### PR TITLE
feat(Restaurant): Create Restaurant Screen and DetailRestaurant screen

### PR DESCRIPTION
**Pull request by hmartiins**
### **Proposed changes**

Create Restaurant screen and Restaurant Detail screen, as I don't know if the Restaurant screen will be refactored in Figma, I gave more focus to the DetailRestaurant screen because it has already gone through the refactoring process, follow the images of how the component was.

<br>

### **Additional Information**
It would be nice to run the app on your device to check if the screen is responsive

### CODE REVIEW

File: src/components/Header/styles.ts [hunk 1]
```diff
@@ -13,7 +13,7 @@ export const Container = styled.View`
 `;
 
 export const IconContainer = styled.TouchableOpacity`
-  right: 50px;
+  right: 200%;
```
Comment (devLeSilverio) [inline src/components/Header/styles.ts:16]: Just to remind you that we should analyze the responsiveness of this icon later.

File: src/screens/Restaurants/components/Session/index.tsx [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+      </SessionHeader>
+
+      <FlatList
+        data={fakeData}
+        keyExtractor={item => String(item.key)}
+        horizontal
+        showsHorizontalScrollIndicator={false}
+        renderItem={({ item }) => (
+          <RestaurantCard
+            onPress={handleNavigateToDetailRestaurant}
```
Comment (devLeSilverio) [inline src/screens/Restaurants/components/Session/index.tsx:69]: Analyze later why the card is not being directed to the details screen :)

**Review by devLeSilverio — CHANGES_REQUESTED** [review]
All right! Congratulations on your efforts! I just left reminders. :)

File: src/components/Header/styles.ts (hunk 1 above)
Comment (hmartiins) [reply src/components/Header/styles.ts:16]: It shouldn't even be there lol, I ended up commit accidentally sry

**Pull request merged**